cookbook/core/containerization/raw_container.py

```diff
@@ -4,71 +4,172 @@
+    inputs=kwtypes(a=float, b=float),
+    outputs=kwtypes(area=float, metadata=str),
+    image="rawcontainers-julia:v1",
+    command=[
+        'julia',
+        'calculate-ellipse-area.jl',
+        '/var/inputs',
+        '/var/outputs',
+    ],
+)
+
+@task
+def report_all_calculated_areas(
+        area_shell: float, metadata_shell: str,
+        area_python: float, metadata_python: str,
+        area_r: float, metadata_r: str,
+        area_haskell: float, metadata_haskell: str,
+        area_julia: float, metadata_julia: str,
+):
+    logger.info(f'shell: area={area_shell}, metadata={metadata_shell}')
+    logger.info(f'python: area={area_python}, metadata={metadata_python}')
+    logger.info(f'r: area={area_r}, metadata={metadata_r}')
+    logger.info(f'haskell: area={area_haskell}, metadata={metadata_haskell}')
+    logger.info(f'julia: area={area_julia}, metadata={metadata_julia}')
 
 # %%
 # As can be seen in this example, ContainerTasks can be interacted with like normal python functions, whose inputs
-# correspond to the declared input variables
+# correspond to the declared input variables. All data returned by
 @workflow
-def raw_container_wf(val1: int, val2: int) -> int:
-    """
-    These tasks can be invoked like simple python methods. But running them locally performs no execution, unless
-    the execution is mocked.
-    """
-    return sum(x=square(val=val1), y=square(val=val2))
+def wf(a: float, b: float):
+    # Calculate area in all languages
+    area_shell, metadata_shell = calculate_ellipse_area_shell(a=a, b=b)
+    area_python, metadata_python = calculate_ellipse_area_python(a=a, b=b)
+    area_r, metadata_r = calculate_ellipse_area_r(a=a, b=b)
+    area_haskell, metadata_haskell = calculate_ellipse_area_haskell(a=a, b=b)
+    area_julia, metadata_julia = calculate_ellipse_area_julia(a=a, b=b)
 
+    # Report on all results in a single task to simplify comparison
+    report_all_calculated_areas(
+        area_shell=area_shell, metadata_shell=metadata_shell,
+        area_python=area_python, metadata_python=metadata_python,
+        area_r=area_r, metadata_r=metadata_r,
+        area_haskell=area_haskell, metadata_haskell=metadata_haskell,
+        area_julia=area_julia, metadata_julia=metadata_julia,
+    )
 
 # %%
-# ContainerTasks cannot really be executed locally as Flytekit is incapable of executing Containers currently.
-# but it is possible to mock the execution.
-if __name__ == "__main__":
-    print(f"Running {__file__} main...")
-    print(
-        f"Running raw_container_wf(val1=5, val2=5) {raw_container_wf(val1=5, val2=5)}"
-    )
+#
+# .. note::
+#   Raw containers cannot be run locally at the moment.
+#
+# Scripts
+# =======
+#
+# The contents of each script mentioned above:
+#
+# calculate-ellipse-area.sh
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#   .. literalinclude::  ../../../../core/containerization/raw-containers-supporting-files/per-language/shell/calculate-ellipse-area.sh
+#       :language: shell
```

Comment:
```suggestion
# .. literalinclude::  ../../../../core/containerization/raw-containers-supporting-files/per-language/shell/calculate-ellipse-area.sh
#    :language: shell
```

Reply:
de-indent to remove vertical quote line on the left side:
![image](https://user-images.githubusercontent.com/2816689/145453865-f691cf8b-b173-45c5-a55b-8f9ff9916689.png)